packages/firebase_auth/firebase_auth_platform_interface/test/mock.dart

```diff
@@ -5,7 +5,7 @@
 import 'package:firebase_auth_platform_interface/firebase_auth_platform_interface.dart';
 import 'package:firebase_auth_platform_interface/src/method_channel/method_channel_firebase_auth.dart';
 import 'package:firebase_core_platform_interface/firebase_core_platform_interface.dart';
-import 'package:firebase_core_platform_interface/test.dart';
+import '../../../firebase_core/firebase_core_platform_interface/test/test.dart';
```

Comment:
I think this is why we previously had flutter_test in dependencies as opposed to dev dependencies so we can share it with other packages. But, I don't think this matters because it will be excluded when publishing to pub.dev. Therefore, relative paths should be fine for other packages

Reply:
Just to double check, I'll see if @Lyokone has any feedback on the matter 😄

Reply:
Hum I'm wondering how it will fix the issue, we have not changed anything in the pubspec.yaml?